game/models.py

```
@@ -582,3 +582,33 @@ def save(self, *args, **kwargs):
+        on_delete=models.CASCADE,
+        related_name="reply_votes"
+    )
+
+    value = models.SmallIntegerField(choices=VOTE_CHOICES)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        unique_together = ("reply", "user")
+
+    def __str__(self):
```

Ruff (0.15.18)

[warning] Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)